pkgs/applications/misc/fusee-interface-tk/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv , fetchFromGitHub , python3 , makeWrapper }: 
+
+let pythonEnv = python3.withPackages(ps: [ ps.tkinter ps.pyusb ]); 
+in stdenv.mkDerivation { 
```

Comment:
Consider using:
```suggestion
buildPythonApplication rec { 
```

Reply:
The reason I'm not using buildPythonApplication is that the package is structured oddly and lacks a setup.py. I plan to try and cobble one together later, but for now the stdenv.mkDerivation should be used. The same applies to the prior and next suggestions.